pkg/gardenlet/controller/shoot/shoot/reconciler.go

```diff
@@ -537,7 +537,9 @@ func (r *Reconciler) updateShootStatusOperationStart(
 		startRotationCA(shoot, &now)
 		startRotationServiceAccountKey(shoot, &now)
 		startRotationKubeconfig(shoot, &now)
-		startRotationSSHKeypair(shoot, &now)
```

Comment:
Can we add tests for this?

Reply:
These functionality is already being checked in e2e test.